cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/CumulusDssFileGenerationListener.java

```diff
@@ -0,0 +1,12 @@
+package mil.army.usace.hec.cumulus.client.controllers;
+
+import mil.army.usace.hec.cumulus.client.model.Download;
+
+public interface CumulusDssFileGenerationListener {
+
+    void downloadStatusUpdated(Download downloadStatus);
```

Comment:
Elapsed time should be provided in the other two events rather than being separate. Additionally, consider using the `java.time.Duration` class for elapsed time.

Reply:
Updated to use Duration